platform/pulpcore/app/models/repository.py

```diff
@@ -125,9 +103,16 @@ class Importer(ContentAdaptor):
     ON_DEMAND = 'on_demand'
     BACKGROUND = 'background'
     DOWNLOAD_POLICIES = (
-        (IMMEDIATE, 'Download Immediately'),
-        (ON_DEMAND, 'Download On Demand'),
-        (BACKGROUND, 'Download In Background'))
+        (IMMEDIATE, 'Sync task is not complete until downloading is finished.'),
+        (ON_DEMAND, 'Sync task updates metadata but does not download content until requested.'),
+        (BACKGROUND, 'Sync task updates metadata and completes. Downloads complete in background.'))
```

Comment:
Suggest something more like:

IMMEDIATE: _Update the repository content and ensures all artifacts are downloaded._

ON_DEMAND: _Update the repository content but no artifacts are downloaded._

BACKGROUND: _Update the repository content and downloads artifacts in the background._

The _tasking_ seems irrelevant to the definition.